.github/workflows/serverless-benchmarks.yml

```diff
@@ -81,6 +84,8 @@ jobs:
             <details>
```

Comment:
Recommendation to collapse the content of the PR comment by default. Or reduce it to 1-3 lines. As of now, it's using an excessive amount or real estate.